src/ApolloData/Query.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useQuery, gql } from '@apollo/client';
+
+interface product {
+  id: number;
+  name: string;
+  image_url: string;
+  description: string;
+  price: number;
+  category: string;
+}
+
```

Comment:
move it to its own file, create a folder of gql queries mutation w/e. and for each gql create a new file